drivers/overlay/peerdb.go

```diff
@@ -22,16 +23,48 @@ type peerEntry struct {
 	eid        string
 	vtep       net.IP
 	peerIPMask net.IPMask
-	inSandbox  bool
 	isLocal    bool
 }
 
+func (p *peerEntry) MarshalDB() peerEntryDB {
+	ones, bits := p.peerIPMask.Size()
+	return peerEntryDB{
+		eid:            p.eid,
+		vtep:           p.vtep.String(),
+		isLocal:        p.isLocal,
+		peerIPMaskOnes: ones,
+		peerIPMaskBits: bits,
+	}
+}
+
+// This the structure saved into the set (SetMatrix), due to the implementation of it
+// the value inserted in the set has to be Hashable so the []byte had to be converted into
+// strings
+type peerEntryDB struct {
+	eid            string
+	vtep           string
+	peerIPMaskOnes int
+	peerIPMaskBits int
```

Comment:
Why is peerIPMask, a `net.IPMask` converted to couple of `int`, while the vtep `net.IP` converted to String ?
Isnt it easy to just marshal `peerIPMask` as a String and that makes it consistent and readable ?

Reply:
it's only convenience, for the net.IPMask looks like the best way to initialize it back is using that decomposition. Ref: https://golang.org/pkg/net/#IPMask

Reply:
Checked again and looks like this can be converted to string and use the ParseCIDR that will return the IPNet that contains the IPMask object internally, will change